map/place_page_info.hpp

```diff
@@ -22,8 +24,24 @@ enum class SponsoredType
   Geochat
 };
 
-namespace place_page
+struct Banner
 {
+  ///  Banner should not outlive the passed metadata.
+  explicit Banner(feature::Metadata const & metadata);
+
+  /// Returns true, if the current time is within the working range of the banner.
+  bool IsActive() const;
+
+  string TitleId() const;
+  string MessageId() const;
+  string IconId() const;
+  string Url() const;
+
+private:
```

Comment:
If you have a private part, please, replace struct by class, and explicitly define a public section.